wcvendors/wcvendors
# WIP: Migrate new Marketplace menu in Admin Dashboard For V3

### All Submissions:






### Changes proposed in this Pull Request:



Added: New Marketplace menu for Vendor in Admin Dashboard and Pagination for orders table

### How to test the changes in this Pull Request:

1. Login as Vendor
2. Go to your site admin dashboard e.g: https://examplesite.com/wp-admin
3. See Marketplace menu hover on Marketplace and view see Orders, Shop Settings, Products, Media 

### TO-DO : 
- Make orders table sortable
### Other information:
